lib/widgets/components/product_card.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+
+class ProductCard extends StatelessWidget {
+  final String image;
+  final String title;
+  final String price;
+
+  const ProductCard({
+    super.key,
+    required this.image,
+    required this.title,
+    required this.price,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    double maxWidth = 168;
+    double maxHeight = 125.516;
+    return Card(
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(11.111),
+          topRight: Radius.circular(11.111),
```

Comment:
Why only top and bottom? On design all edges have border radius, and when I run app on simulator neither edge have border radius?